monitoring_test.go

```diff
@@ -86,8 +86,8 @@ func TestUnmarshalMonitoringJobs(t *testing.T) {
 		t.Error("Status is not up")
 	}
 	r := j.Rules[0]
-	assert.Equal(t, r.Key, "rtt", "RTT rule key is wrong")
-	assert.Equal(t, r.Value.(float64), float64(100), "RTT rule value is wrong")
+	// assert.Equal(t, r.Key, "rtt", "RTT rule key is wrong")
```

Comment:
I wanted to remove any non std lib dependencies. This was the only one I saw(are there more?). I think we will be able to still write the tests without `testify/assert`, but I should jump on that now. Think we should leave it in?